README.md

```diff
@@ -4,21 +4,108 @@
 ## What is ocdev?
 OpenShift Command line for Developers
 
+## Pre-requisites
+- OpenShift version 3.7 and up
+
+To use ocdev you need access to OpenShift instance and OpenShift CLI installed on your local machine (`oc` should be in your $PATH).
+
+
+### OpenShift instance
+You can use [Minishift](https://docs.openshift.org/latest/minishift/index.html) to get a local instance of OpenShift. However ocdev can be used with any instance of OpenShift.
+
+### OpenShift CLI 
+There are different ways to install OpenShift CLI. 
+Please follow [OpenShift documentation](https://docs.openshift.org/latest/cli_reference/get_started_cli.html#installing-the-cli).
+
+
 ## Installation
-You can [download](https://dl.bintray.com/ocdev/ocdev/latest/) latest binaries (build automatically from current master branch).
+You can download latest master builds from [Bintray](https://dl.bintray.com/ocdev/ocdev/latest/) or 
+builds for released versions from [GitHub releases page](https://github.com/kadel/ocdev/releases).
+
```

Comment:
-1 for redirecting to personal repository for releases. I would suggest you to do an alpha release the main repository if you want to mention github release or we can just remove it from the readme for now. We can add it later when the release will be done.

Reply:
that is a typo.
the link should be https://github.com/redhat-developer/ocdev/releases
thank for noticing